django/contrib/auth/middleware.py

```diff
@@ -23,6 +24,30 @@ def process_request(self, request):
         request.user = SimpleLazyObject(lambda: get_user(request))
 
 
+class LoginRequiredAuthenticationMiddleware(AuthenticationMiddleware):
+    """
+    Middleware that force all views to require athentication by default.
+
+    Views that have @login_not_required decorator or LoginNotRequiredMixin mixin
+    will be able to pass through without this validation. Otherwise, it will
+    direct user to the login.
+    """
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        # If it's authenticated user we don't have to do anything.
```

Comment:
Please avoid "we" in comments. (See [Coding style guidelines](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/))